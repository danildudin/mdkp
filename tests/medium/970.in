28 6 0
267 203 821 584 48 118 468 311 927 440 560 966 782 727 43 826 437 274 190 615 839 473 300 904 214 781 320 961 
744 163 533 413 949 672 293 806 847 526 104 170 649 987 431 815 247 104 880 25 258 926 976 474 200 150 737 154 
665 426 602 34 234 553 404 694 332 600 705 503 125 724 391 417 584 71 377 731 203 54 404 985 663 779 394 463 
106 874 58 142 304 675 933 165 647 300 372 578 664 842 514 933 205 584 23 344 756 825 162 974 401 631 299 764 
962 40 200 205 147 705 856 720 250 862 523 254 966 96 13 998 747 979 259 24 770 232 11 746 623 319 227 229 
562 446 821 528 210 642 666 521 157 587 189 850 641 166 944 799 394 123 344 657 389 214 413 752 73 890 606 381 
670 167 57 556 651 947 420 209 611 488 587 305 174 62 590 674 421 954 302 337 94 633 425 970 695 146 933 689 
8942 6364 9620 6786 8313 5571 
