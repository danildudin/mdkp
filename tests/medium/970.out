10400
15
2 3 6 8 9 11 12 13 15 18 19 20 21 25 27 
